<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicSearchDialog</class>
 <widget class="QDialog" name="MusicSearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/image/background/searchbackground.webp);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-30</y>
      <width>700</width>
      <height>700</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>700</width>
      <height>700</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>700</width>
      <height>700</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/image/background/searchbackground.png);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="SearchlineEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="SearchButton">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>110</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>35</width>
       <height>35</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>关于</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="imagefile.qrc">
       <normaloff>:/image/image/image/search2.png</normaloff>:/image/image/image/search2.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>歌曲搜索</string>
     </property>
    </widget>
    <widget class="QListWidget" name="nameListWidget">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>180</y>
       <width>561</width>
       <height>241</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 50);
</string>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="downloadButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>下载</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="imagefile.qrc"/>
 </resources>
 <connections/>
</ui>
